docs/docs/cmd/setup.md

```diff
@@ -0,0 +1,81 @@
+# setup
+
+Sets up CLI for Microsoft 365 based on your preferences
+
+## Usage
+
+```sh
+m365 setup [options]
+```
+
+## Options
+
+`--interactive`
+: Configure CLI for Microsoft 365 for interactive use without prompting for additional information
+
+`--scripting`
+: Configure CLI for Microsoft 365 for use in scripts without prompting for additional information
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The `m365 setup` command is an interactive wizard that helps you configure the CLI for Microsoft 365 for your needs. It will ask you a series of questions and based on your answers, it will configure the CLI for Microsoft 365 for you.
```

Comment:
The word 'interactive' is used multiple times on this page in multiple connotations:

1. The command itself can be 'interactive' (It's an interactive wizard)
2. The command can configure the CLI to be 'interactive'

The term becomes slightly ambiguous like this. I suggest we remove the word 'interactive' before wizard to avoid this.

Reply:
Good perspective. Let me adjust that.